homeassistant/components/unifiprotect/switch.py

```diff
@@ -31,12 +31,10 @@ class ProtectSwitchEntityDescription(ProtectSetableKeysMixin, SwitchEntityDescri
 
 
 def _get_is_highfps(obj: Any) -> bool:
-    assert isinstance(obj, Camera)
     return bool(obj.video_mode == VideoMode.HIGH_FPS)
 
 
 async def _set_highfps(obj: Any, value: bool) -> None:
```

Comment:
```suggestion
async def _set_highfps(obj: Any, value: Camera) -> None:
```

Reply:
Pretty fure you mean to make `obj` a Camera.